plumber/asset/model.py

```diff
@@ -64,10 +64,12 @@ def import_model(self, model: Model, collection: Collection) -> None:
             if material is None:
                 material_data = get_unknown_material()
             else:
+                material_original_name = material
                 material = truncate_name(material)
                 material_data = bpy.data.materials.get(material)
                 if material_data is None:
                     material_data = bpy.data.materials.new(material)
+                material_data['path_id'] = material_original_name
```

Comment:
This can probably be inside the if above, so that the material "path id" is only stored when the material is created and not unnecessarily every time it is used

Reply:
that sounds better, i'll fix that

Reply:
Sorry, didn't catch this earlier, but I think this can be actually removed completely since the path_id is already imported in the material.py file.